transform/mattermost-analytics/macros/rudderstack/union_relations.sql

```diff
@@ -0,0 +1,128 @@
+{%- macro union_relations(relations, column_override=none, include=[], exclude=[], source_column_name='_dbt_source_relation', where=none) -%}
```

Comment:
No need to add this. This macro originates from [dbt-utils](https://github.com/dbt-labs/dbt-utils) package. This package is already part of the project, so you can use the [union_relations](https://github.com/dbt-labs/dbt-utils#union_relations-source) macro from the package directly.

Here's [an example](https://github.com/mattermost/mattermost-data-warehouse/blob/master/transform/mattermost-analytics/models/intermediate/data_eng/event_registry/int_events_aggregated_to_date.sql) on how it's used in another filemodelin the project.